db/app.js

```diff
@@ -1,13 +1,20 @@
 const express = require("express"),
   { getAllCategories } = require("./controllers/categories.controllers.js"),
   { getAllUsers } = require("./controllers/users.controllers.js"),
-  { getReviewById } = require("./controllers/reviews.controllers.js"),
+  {
+    getReviewById,
+    incrementVotes,
+  } = require("./controllers/reviews.controllers.js"),
   app = express();
 
+app.use(express.json());
+
 app.get("/api/categories", getAllCategories);
 app.get("/api/users", getAllUsers);
 app.get("/api/reviews/:review_id", getReviewById);
 
+app.patch("/api/reviews/:review_id", incrementVotes);
+
```

Comment:
This is a comment for line 25, I would move this to above your error handling as it isn't strictly an error handling function and is related to the endpoints

Reply:
ah ok, after that am i good to merge?